src/index.js

```diff
@@ -1 +1,16 @@
-'use strict';

const scale = require('./mode');
const chord = require('./chord');
const clip = require('./clip');
const pattern = require('./pattern');
const midi = require('./midi');
const scales = require('./modes');

let modes = Object.keys(scales);

// Allow scale to be denoted by mode as well
module.exports = {mode: scale, scale, chord: chord.getChord, listChords: chord.listChords, modes, scales: modes, clip, pattern, midi};
\ No newline at end of file
+'use strict';
+const scale = require('./mode');
+const chord = require('./chord');
+const clip = require('./clip');
+const pattern = require('./pattern');
+const midi = require('./midi');
+const scales = require('./modes');
+function setMiddle(octaveIndex){
```

Comment:
At a interface level I m ok to do this `scribble.setMiddleC(Number)` but lets try to come up with a better way to put it together at the code level. Defining `setMiddle` in 2 different files seems counterintuitive. 

I ll give this some more thought too.